manipulation/ManipulationPlanRequest manipulation_plan_request

---

moveit_task_constructor_msgs/Solution manipulation_plan_reponse
